documentation/logo.md

```diff
@@ -81,12 +82,36 @@ show :y
 ERROR
 ```
 
-## Laskutoimitukset
 
-Vain FLOAT-tyyppisillä muuttujilla. Operaatiot ovat `+`, `-`, `*` ja `/`.
+## Mathematical operations
+
+Mathematical operations are possible to do only with a variable of type `FLOAT`. Operations that are possible are `+`, `-`, `*` and `/`
+
+## Relational operations
+
+Variables of type `FLOAT` are comparable with each other with the operators `<`, `<=`, `>`, `>=`, `=` and `<>`.
+
+`STRING` variables can be compared with each other only by using the operators `=` and `<>`.
 
-## Vertailuoperaatiot
+## Case sensitivity
 
-FLOAT-tyyppisiä muuttujia voidaan vertailla operaattoreilla `<`, `<=`, `>`, `>=`, `=` ja `<>`. 
+Variables, commands, function names and other referenced types of information are all case insensitive. For example:
```

Comment:
saattais olla helpompaa määrittää negaation kautta ts. vain show roskat on ei ole case insensitiivisiä tms. referenced types of information on aika jämäkkä ilmaisu